qa/sc_backward_transfer.py

```diff
@@ -311,5 +311,5 @@ def run_test(self):
 
 
 if __name__ == "__main__":
-    // TODO add nonCeasing=false argument for sharing tests
+    # TODO add nonCeasing=false argument for sharing tests
```

Comment:
See the approach from Sergiy on similar extensions: https://github.com/HorizenOfficial/Sidechains-SDK/pull/583/files#diff-b17e173f5d2e402b9b9be183e81c3e0a3c176e184f82d85a2bc85f5102872240R146
Let's do the same